examples/sandbox/README.md

```diff
@@ -0,0 +1,29 @@
+# Sandbox
+
+This example is a sandbox for local development. It sets up a basic page
+using Search UI.
+
+It is wired up to use the local source of the search-ui libraries as dependencies via lerna.
+This means as you update library code you can see it updated live in this project.
+
+It is set up to use a pre-configured Elastic App Search engine.
```

Comment:
It would be nice to have some information about the engine and the dataset.

Considering the engine config: Since this is just a testbed for the Search UI, I'm not sure it's a place for people to change the engine config, what do you think?

Reply:
@brianearwood I provided a bit more information about what the data set is.

> I'm not sure it's a place for people to change the engine config, what do you think?

I'm not quite sure what you're asking. But if you're developing locally, I think it would be pretty reasonable to change the engine config in whatever way you'd need. You might point it to an entirely different engine, or you might change the parameters... you just wouldn't be able to commit those changes back to the repository.

Did does that makes sense?

Reply:
That makes sense.

I was wondering if we should point out with additional copy how people could easily swap out the default test engine credentials with their own to make the sandbox more meaningful for their use-case.

Reply:
Ah, great point. And actually, I think I could probably move the credentials to a .env file, so that you can update the engine and not have to worry about accidentally commiting your changes 👍

Reply:
Just updated the instructions to include details about configuring a .env file.

And now that I'm thinking about this, once Reference UI is converted to use Search UI, I should totally just make the Sandbox an instance of the Reference UI.

That way you can configure much more than just the engine.